src/react-components/hub-create-panel.js

```diff
@@ -194,15 +194,27 @@ class HubCreatePanel extends Component {
                   />
                   <div className="create-panel__form__environment__picker__labels">
                     <div className="create-panel__form__environment__picker__labels__header">
-                      <span className="create-panel__form__environment__picker__labels__header__title">
-                        {environmentTitle}
-                      </span>
+                      {meta.url ? (
+                        <a
+                          href={meta.url}
+                          target="_blank"
+                          rel="noopener noreferrer"
+                          className="create-panel__form__environment__picker__labels__header__title"
+                        >
+                          {environmentTitle}
+                        </a>
+                      ) : (
+                        <span className="create-panel__form__environment__picker__labels__header__title">
+                          environmentTitle
+                        </span>
+                      )}
                       {environmentAuthor &&
                         environmentAuthor.name &&
                         (environmentAuthor.url ? (
                           <a
                             href={environmentAuthor.url}
                             target="_blank"
```

Comment:
I'd remove the `target` (for [these reasons](https://css-tricks.com/use-target_blank/))